### DISCUSSION TITLE
Error 503

### DISCUSSION THREAD

**Discussion by darth2602** [lllyasviel/Fooocus#2792] [Q&A] [answered]
for a few hours now I've been unable to launch Fooocus

Collecting pygit2==1.12.2
  Downloading pygit2-1.12.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (4.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.9/4.9 MB 17.4 MB/s eta 0:00:00
Requirement already satisfied: cffi>=1.9.1 in /usr/local/lib/python3.10/dist-packages (from pygit2==1.12.2) (1.16.0)
Requirement already satisfied: pycparser in /usr/local/lib/python3.10/dist-packages (from cffi>=1.9.1->pygit2==1.12.2) (2.22)
Installing collected packages: pygit2
Successfully installed pygit2-1.12.2
/content
Cloning into 'Fooocus'...
remote: Enumerating objects: 5849, done.
remote: Counting objects: 100% (14/14), done.
remote: Compressing objects: 100% (13/13), done.
remote: Total 5849 (delta 1), reused 7 (delta 1), pack-reused 5835
Receiving objects: 100% (5849/5849), 32.67 MiB | 27.60 MiB/s, done.
Resolving deltas: 100% (3370/3370), done.
/content/Fooocus
Already up-to-date
Update succeeded.
[System ARGV] ['entry_with_update.py', '--preset', 'realistic', '--share', '--always-high-vram']
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Fooocus version: 2.3.1
Error checking version for torchsde: No package metadata was found for torchsde
Installing requirements
Loaded preset: /content/Fooocus/presets/realistic.json
[Cleanup] Attempting to delete content of temp dir /tmp/fooocus
[Cleanup] Cleanup successful
Downloading: "https://huggingface.co/lllyasviel/misc/resolve/main/xlvaeapp.pth" to /content/Fooocus/models/vae_approx/xlvaeapp.pth

Traceback (most recent call last):
  File "/content/Fooocus/entry_with_update.py", line 46, in <module>
    from launch import *
  File "/content/Fooocus/launch.py", line 132, in <module>
    config.default_base_model_name, config.checkpoint_downloads = download_models(
  File "/content/Fooocus/launch.py", line 98, in download_models
    load_file_from_url(url=url, model_dir=config.path_vae_approx, file_name=file_name)
  File "/content/Fooocus/modules/model_loader.py", line 25, in load_file_from_url
    download_url_to_file(url, cached_file, progress=progress)
  File "/usr/local/lib/python3.10/dist-packages/torch/hub.py", line 620, in download_url_to_file
    u = urlopen(req)
  File "/usr/lib/python3.10/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.10/urllib/request.py", line 525, in open
    response = meth(req, response)
  File "/usr/lib/python3.10/urllib/request.py", line 634, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.10/urllib/request.py", line 563, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 503: Service Temporarily Unavailable

**Comment by ILikeToasters** [comment, marked as answer]
HTTP Error 503: Service Temporarily Unavailable - huggingface is having issues it seems so it can not download what it needs at the moment.

**Reply by mashb1t** [reply]
@darth2602 seems to work again. Please relaunch Fooocus so it can retrigger the download.

**Comment by saboreje1301** [comment]
[System ARGV] ['entry_with_update.py', '--share', '--always-high-vram']
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Fooocus version: 2.3.1
Error checking version for torchsde: No package metadata was found for torchsde

I have the same problem, I've try to access for a week and I haven't been able to
